### DISCUSSION TITLE
Release v0.19.2 Beta

### DISCUSSION THREAD

**Discussion by PrestonN** [FreeTubeApp/FreeTube#4709] [Announcements]
This is a small release for now. Mostly fixes the issues that has recent occurred with the Local API. We have a very large release for everyone soon so I hope that everyone looks forward to it.

## Fixes

- Updated Local API to the latest version of YouTube.js which fixes the recent API errors for both watching and searching for videos (#4707)
- Fixed errors occasionally cropping up when subscriptions are refreshed on the Local API (#4543)
- Fixes the channel about tab missing information on the Local API (#4370)

<hr /><em>This discussion was created from the release <a href='https://github.com/FreeTubeApp/FreeTube/releases/tag/v0.19.2-beta'>Release v0.19.2 Beta</a>.</em>

**Comment by rks1975** [comment]
Videos Loading slowly/laggy than 19.1.

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
Hi @rks1975 is the Local API set as your Primary API?

And

Is the toggle Proxy Through Invidious **Disabled** in the player settings?

**Comment by vdbhb59** [comment]
Videos are buffering a lot and pausing way too much. Any video above 720P on 2.4 is fine but on 5G is lagging. Weird.

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
Hi @vdbhb59 is the Local API set as your Primary API?

And

Is the toggle Proxy Through Invidious **Disabled** in the player settings?

Also can you provide a screenshot of the About tab?

**Reply by vdbhb59** [reply]
Hi. Yes for both.

![image](https://github.com/FreeTubeApp/FreeTube/assets/60728004/701e7f1a-d201-4274-b8c3-87ee7da0129a)
![image](https://github.com/FreeTubeApp/FreeTube/assets/60728004/fc685caa-49fa-4ae7-b85c-fb73bff688f1)
![image](https://github.com/FreeTubeApp/FreeTube/assets/60728004/96d865dd-34a7-4f0c-bc14-59f333ece5e1)

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
On what system is this installed on?

**Reply by vdbhb59** [reply]
Windows 11 as of now. December build.

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
Then why did you mentioned 5G do you mean WIFI 5Ghz?

Or do you mean 5G in like the mobile network?

**Reply by vdbhb59** [reply]
Sorry should have mentioned clearly.
5G WiFi connection and 2.4G Wi-Fi connection.

**Comment by Ruygazunc** [comment]
Was having a lot of problems with Local API the last few days but this update (19.2) seems to to have cleared it up. Thanks!

**Comment by Ygarr** [comment]
How about apk version? NewPipe alternative? There was "cordova version" a lont time ago......

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
Check releases on https://github.com/MarmadileManteater/FreeTubeCordova

**Reply by NGP39** [reply]
FreeTubeCordova is better than NewPipe? it's worth to test it on Android instead of NewPipe?

**Reply by vdbhb59** [reply]
Both are good with few changes. I use both and yes both are available and updated regularly.

**Comment by Backhage** [comment]
Cool cool cool. Will there be a new Snap made available for this release as well?

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
@capecrusader-121 any update on this?

**Reply by ghost** [reply]
@efb4f5ff-1298-471a-8973-3d47447115dc  Yes i will publish it in the next few hours, cheers.

**Reply by Backhage** [reply]
Super! Thanks a bunch.

**Comment by M-Hamano16** [comment]
One thing I notice- I love freetube, but there is a problem.  I use it on several PCs, all of my PCs use linux mint.  I use the "suspend" feature frequently.  I will set up a few videos to play, and then set the computer to auto suspend in a half hour say after I fall asleep.  When I un-suspend the computer it's like Freetube needs to be re-loaded to realize it has internet again.  If I don't do this, the videos will just play until the point where it loaded data and then it will just spin as if there is no connection.  I can fix this by just clicking back and forward then again.  It's not a big deal but still a minor pain.  I have tested this behavior on several computers- and it seems to be consistent.  Is this something you can help with?

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
There is an open issue for this but nothing you can do about this at the moment.

https://github.com/FreeTubeApp/FreeTube/issues/4159
